cmd/manager/main.go

```diff
@@ -94,6 +96,7 @@ func main() {
 		LivenessEndpointName:   "/livez",
 		LeaderElection:         true,
 		LeaderElectionID:       "hostpath-provisioner-operator-lock",
+		WebhookServer:          cryptopolicy.GetWebhookServerSpec(),
```

Comment:
I don't think the reconcile loop has run here yet, so we haven't configured the environment variables from the cluster yet. So when it gets the webhook.server, there is no way it has located the right information. You should probably build a client right after getting the config, use that to get the api server and set environment variables, so it can be picked up when the mgr starts all the servers.

Reply:
I missed the `cfg.GetConfigForClient` which makes it all work, ignore